### PR TITLE
Permanently disable SA1603 (DocumentationMustContainValidXml)

### LINKED ISSUE #1291: Proposal: Permanently disable rule SA1603 (DocumentationMustContainValidXml)
This rule is a duplication of compiler warning [CS1570](https://msdn.microsoft.com/en-us/library/c20zzdxx.aspx), which means we're slowing down the build but not gaining any functionality from it.

### PR DESCRIPTION
**Pull request by sharwell**
Fixes #1291

### CODE REVIEW

File: StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1603DocumentationMustContainValidXml.cs [hunk 1]
```diff
@@ -35,7 +31,7 @@ public class SA1603DocumentationMustContainValidXml : DiagnosticAnalyzer
         private const string HelpLink = "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1603.md";
 
         private static readonly DiagnosticDescriptor Descriptor =
-            new DiagnosticDescriptor(DiagnosticId, Title, MessageFormat, AnalyzerCategory.DocumentationRules, DiagnosticSeverity.Warning, AnalyzerConstants.EnabledByDefault, Description, HelpLink);
+            new DiagnosticDescriptor(DiagnosticId, Title, MessageFormat, AnalyzerCategory.NamingRules, DiagnosticSeverity.Warning, AnalyzerConstants.DisabledByDefault, Description, HelpLink, WellKnownDiagnosticTags.NotConfigurable);
```
Comment (pdelvo) [inline StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1603DocumentationMustContainValidXml.cs]: This is not a naming rule

**Comment by pdelvo** [comment]
I wonder if we should also update the docs for the rules

**Comment by sharwell** [comment]
> I wonder if we should also update the docs for the rules

Yes, I think documenting the disabled rules would be a nice addition to #359. I'm not sure yet what the best way to do that is.

File: StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1603DocumentationMustContainValidXml.cs (hunk 1 above)
Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1603DocumentationMustContainValidXml.cs]: I amended the previous commit to correct this.

**Comment by pdelvo** [comment]
:+1:

**Pull request merged**